test/routes/routes.test.js

```diff
@@ -78,19 +71,23 @@ const createTests = (token) => {
 						headers,
 					};
 
-					if (!isOnIgnoreList(route, method)) {
+					if (!isOnIgnorelist(route, method, ignorelist)) {
 						const status = await request(options)
 							.then((res) => res.statusCode)
 							.catch((error) => error.statusCode);
 
-						expect(status).to.satisfies((s) => acceptedResults(s, route, method));
+						if (isOnWhitelist(route, method, whitelist)) {
+							expect(status).to.equal(whitelist[route][method]);
+						} else {
+							expect(status).to.be.oneOf([401, 405]);
+						}
 					}
 				});
 			}
 		});
 	}
 };
 
-describe('Call routes without jwt', () => createTests());
-describe('Call routes with empty jwt', () => createTests(''));
-describe('Call routes with invalid jwt', () => createTests('eyJhbGciOiJIUzI1NiIsInR5cCI6ImFjY2VzcyJ9.eyJhY2Nv'));
+describe('Call routes without jwt', () => createTests(undefined, whitelistNoJwt, ignorelistNoJwt));
+describe('Call routes with empty jwt', () => createTests('', whitelistNoJwt, ignorelistNoJwt));
+describe('Call routes with invalid jwt', () => createTests('eyJhbGciOiJIUzI1NiIsInR5cCI6ImFjY2VzcyJ9.eyJhY2Nv', whitelistInvalidJwt, ignorelistInvalidJwt));
```

Comment:
Codacy found an issue: [This line has a length of 156. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5673434)